.go-version

```diff
@@ -1 +1 @@
-1.16.0
```

Comment:
Is this file even used for anything anymore?

Reply:
This auto-updated to 1.18 when I changed to 1.18 using goenv, so I guess it must be used in the goenv ecosystem